api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -191,6 +203,63 @@ private FirecloudWorkspaceId generateFirecloudWorkspaceId(String namespace, Stri
     return ResponseEntity.ok(createdWorkspace);
   }
 
+  @Override
+  public ResponseEntity<WorkspaceOperation> createWorkspaceAsync(Workspace workspace) {
+    // Basic request validation.
+    validateWorkspaceApiModel(workspace);
+    getLiveCdrVersionId(workspace.getCdrVersionId());
+
+    // TODO: enforce access level check here? Not strictly necessary, but may make sense as
+    // belt/suspenders check.
+
+    Timestamp now = new Timestamp(clock.instant().toEpochMilli());
+    DbWorkspaceOperation operation =
+        workspaceOperationDao.save(
+            new DbWorkspaceOperation()
+                .setCreatorId(userProvider.get().getUserId())
+                .setStatus(DbWorkspaceOperationStatus.PENDING)
+                .setCreationTime(now)
```

Comment:
Since you have `@CreatedDate`, `@LastModifiedDate`, these values should be populated automatically for you